packages/thumbprint-atomic/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+### Added
+
+-   [Minor] Adding IE/Edge selectors to `box-shadow` to bring visual parity with modern browsers. (#476)
```

Comment:
```suggestion
-   [Patch] Adding IE/Edge selectors to `box-shadow` to bring visual parity with modern browsers. (#476)
```

Changing because it's a bug fix.